automated-comments/javascript/general/prefer_unprefixed_underscore_parameters.md

```diff
@@ -0,0 +1,5 @@
+Unlike other languages, `_parameter` does not signify a *private* variable.
```

Comment:
```suggestion
Unlike other languages, `_parameter ` does not signify a *private* parameter.
```

Should it be parameter instead of variable?

Reply:
Ah. It should be `_identifier does not signify a *private* function, parameter or variable`

Reply:
That sounds better!

Reply:
```suggestion
Unlike other languages, `_identifier` does not signify a **private** function, parameter, or variable.
```